zap/src/main/java/org/parosproxy/paros/common/AbstractParam.java

```diff
@@ -168,7 +167,7 @@ protected String getString(String key, String defaultValue) {
      * @since 2.7.0
      */
     protected static void logConversionException(String key, ConversionException e) {
-        logger.warn("Failed to read '" + key + "'", e);
+        logger.warn("Failed to read [{}]", key, e);
```

Comment:
Were the single quotes replaced with square brackets intentionally?

Reply:
Yes it was fine purposefully. I remembered having trouble with single quotes at one point. But, now that you’ve got me thinking about it I wonder if that was logging or Messages.properties ??

@psiinon @thc202 does that ring a bell with either of you.

Reply:
`Messages.properties`

Reply:
Okay, I'll restore the single quotes in these then.

Thank you!